pkg/reconciler/openshift/tektonhub/extension.go

```diff
@@ -168,9 +173,165 @@ func (oe openshiftExtension) PreReconcile(ctx context.Context, tc v1alpha1.Tekto
 }
 
 func (oe openshiftExtension) PostReconcile(ctx context.Context, tc v1alpha1.TektonComponent) error {
+	th := tc.(*v1alpha1.TektonHub)
+	consoleCLILS := metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			v1alpha1.InstallerSetType: ConsoleHubLinkInstallerSet,
+		},
+	}
+
+	consoleHubLinkLabelSelector, err := common.LabelSelector(consoleCLILS)
+	if err != nil {
+		return err
+	}
+
+	exist, err := checkIfInstallerSetExist(ctx, oe.operatorClientSet, common.TargetVersion(th), consoleHubLinkLabelSelector)
+	if err != nil {
+		return err
+	}
+
+	if !exist {
+		hubConsoleLinkManifest := oe.manifest.Append()
+		if err := applyHubConsoleLinkManifest(&hubConsoleLinkManifest); err != nil {
+			return err
+		}
+
+		if err := consoleLinkTransform(ctx, &hubConsoleLinkManifest, th.Status.GetUiRoute()); err != nil {
+			return err
+		}
+
+		if err := createInstallerSet(ctx, oe.operatorClientSet, th, hubConsoleLinkManifest,
```

Comment:
If the console link instance is not created should we mark the status of post reconciler with an error

Reply:
if the installerset error has any error we should return error and the logic outside the postReconcile call in the main reconciler should be  handling that error